src/app/models/form.server.model.ts

```diff
@@ -197,7 +197,26 @@ const EncryptedFormSchema = new Schema<IEncryptedFormSchema>({
     type: String,
     required: true,
   },
-
+  emails: {
+    type: [
+      {
+        type: String,
+        trim: true,
+      },
+    ],
+    set: transformEmails,
+    validate: [
+      (v: string[]) => {
+        if (!Array.isArray(v)) return false
+        if (v.length === 0) return true
+        return v.every((email) => validator.isEmail(email))
+      },
+      'Please provide valid email addresses',
+    ],
+    // Mongoose v5 only checks if the type is an array, not whether the array
```

Comment:
We're no longer using mongoose v5, could this be removed?

Reply:
Updated to say Mongoose v6!